src/main/webapp/app/lecture/lecture-attachments.component.ts

```diff
@@ -29,7 +29,8 @@ export class LectureAttachmentsComponent implements OnInit, OnDestroy {
     erroredFile?: Blob;
     errorMessage?: string;
 
-    readonly allowedFileExtensions = FILE_EXTENSIONS.join(', ');
+    readonly allowedFileExtensions = FILE_EXTENSIONS.map((ext) => '.' + ext).join(',');
+    readonly allowedFileExtensionsString = FILE_EXTENSIONS.join(', ');
```

Comment:
I don’t think `…String` is a good name. Maybe `acceptedFileExtensionsFileBrowser` and `allowedFileExtensions` with a comment before the first one why two separate ones are needed would be better?